ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -1,10 +1,12 @@
-from datetime import datetime
-from dateutil.parser import parse
 import logging
-import isodate
+from datetime import datetime
+
 import ckan.plugins.toolkit as tk
```

Comment:
Maybe it is nicer to have a more "speaking" name instead of **tk** ?

Reply:
We usually use `tk`, so I think it's okay to keep it like this here.